src/main/java/stringCalculator/domain/Number.java

```diff
@@ -0,0 +1,39 @@
+package stringCalculator.domain;
+
+import java.util.Objects;
+
+public class Number {
+  private int number;
```

Comment:
숫자를 Value Object로 다루셨군요! 👍 
좀 더 완벽에 가까운 VO로 다루기 위해 아래와 같이 수행한다면 어떤 장점이 있을까요 :) ?
```suggestion
public final class Number {
  private final int number;
```

Reply:
변수에 final 붙여주면 재할당이 불가능하고 클래스에 final 을 붙이면 상속이 불가능한 것으로 알고 있어요!  final 을 붙여서 불변객체로 만들면, 오로지 Number 라는 래퍼 클래스를 통해서만 변경이 가능해 져서 number 변수가 언제 어디서 바뀔지 불안해 하지 않아도 되겠네요 :)